tests/shared/src/test/scala/org/typelevel/toolkit/ScalaCliProcess.scala

```diff
@@ -61,7 +61,7 @@ object ScalaCliProcess {
       )
       .evalTap { path =>
         val header = List(
-          s"//> using scala ${BuildInfo.scalaVersion}",
+          s"//> using scala ${BuildInfo.scalaBinaryVersion}",
```

Comment:
In practice, a user is not going to specify the full Scala version. Most likely they will specify nothing (equivalent to `3`) or `2` (equivalent to `2.13`) or `2.13`. So actually we should check that our toolkit works with whatever default Scalas that scala-cli is shipping.

Reply:
I usually specify the version, but I got your point.